models/index.js

```diff
@@ -1,27 +1,33 @@
-'use strict';
+"use strict";
```

Comment:
is this just prettier or did we change anything in this file?

Reply:
yes this was just prettier, I din't change anything here.